### ISSUE TITLE
Offline prediction code for CILRS

### ISSUE THREAD

**Issue by Flora279** [felipecode/coiltraine#46]
Hi,
I have been looking around the code, however, I was not able to find where in the code we can get predictions (steer, throttle, brake) for a single sample (rgb+speed+control direction) input. Also, I cannot find where the speed branch is used to adjust the throttle for avoid stopping when the car shouldn't. 

May I have some guidance regarding this?

**Comment by Flora279** [comment]
Hi,
Sorry for disturbance again.

When I read through the resnet34imnet10-nospeed.yaml file, I found when training the model you also include the speed branch in the loss function to update the nospeed model (since weight for speed branch is 0.05). 

Am I correct about this?

If yes, then the training process & data & model architecture for resnet34imnet10-nospeed.yaml and resnet34imnet10S1.yaml would be the same?

Looking forward to your reply.

**Comment by Flora279** [comment]
@felipecode @markus-hinsche @AdamGleave 
Hi, apologize for contacting again, but this is actually really confusing me.
1. If CILR corresponding to the model without speed branch, then why we also give 0.05 weight to the speed branch in the training loss in the file resnet34imnet10-nospeed.yaml? Also, after loading the resnet34imnet10-nospeed checkpoint file, I also found learned parameters for speed branch.
2. In the CILRS resnet34imnet10S1.yaml file with speed branch, for the simulation related parameter why we still set 'AVOID-STOPPING = False'?

Any clarification on these would be fantastic, looking forward to your reply.
